main.nf

```diff
@@ -268,114 +200,377 @@ ${summary.collect { k,v -> "            <dt>$k</dt><dd><samp>${v ?: '<span style
    return yaml_file
 }
 
-
->>>>>>> TEMPLATE
 /*
  * Parse software version numbers
  */
 process get_software_versions {
-<<<<<<< HEAD
     validExitStatus 0,2
-=======
->>>>>>> TEMPLATE
 
     output:
     file 'software_versions_mqc.yaml' into software_versions_yaml
 
     script:
     """
-<<<<<<< HEAD
-    echo $params.version > v_pipeline.txt
+    echo $params.pipelineVersion > v_pipeline.txt
     echo $workflow.nextflow.version > v_nextflow.txt
     multiqc --version > v_multiqc.txt
-    atropos | head -2 | tail -1 > v_atropos.txt
-=======
-    echo $manifest.pipelineVersion > v_pipeline.txt
-    echo $workflow.nextflow.version > v_nextflow.txt
     fastqc --version > v_fastqc.txt
-    multiqc --version > v_multiqc.txt
->>>>>>> TEMPLATE
+    fastp --version > v_fastp.txt
+    megahit --version > v_megahit.txt
```

Comment:
`refinem`, `metabat` are missing here (and others ?)

Reply:
`refinem`, `checkm` and `metabat` do not have a `--version` flag. Should I go through the hassle of parsing the help messages?

Reply:
Unfortunately, yes. This is what we do for other tools without a proper `-version` or other info switch :-(

Reply:
could we not fetch the version info from conda? (Sorry if you folks had that discussion before on slack or another pipeline)

Reply:
Better to just parse the version with a regex. It should be pretty simple, no? I usually just dump the output into https://regex101.com/ and play around until I get a pattern that fetches what I want.

Reply:
I much prefer getting the version information from the tool itself though. There are all kinds of weird edge cases where strange stuff could happen in the installation. The only way to be really sure about the version is to query it using the same command line command that is used to run the actual analysis.

Reply:
Me too - we just had the discussion about tools that do not provide *any* version information (dupRadar in rnaseq for example, https://github.com/nf-core/rnaseq/issues/112). For all other tools I also think this is the safest way to get exactly the utilized version into the report 👍

Reply:
Alright! thanks both for your input